documentation/sphinx/source/client-testing.rst

```diff
@@ -0,0 +1,229 @@
+###############
```

Comment:
This guide only works if you set the cmake option`WITH_JAVA_WORKLOAD`, right? I think it'd be good to mention that here.

Reply:
Good point. Added a short section in the documentation